Octokit.Tests.Integration/Clients/RepositoryHooksClientTests.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Octokit.Tests.Integration.fixtures;
+using Xunit;
+
+namespace Octokit.Tests.Integration.Clients
+{
+    public class RepositoryHooksClientTests
+    {
+        [Collection("Repositories Hooks Collection")]
```

Comment:
:grey_question: we've turned off parallelization here as part of running the integration tests, so these guards aren't necessary. Do you think they're useful to have anyway when running these hooks tests?